### PR TITLE
Replace moreOptions section and toolbar with a pulldown menu in StopViewController

### PR DESCRIPTION
**Pull request by ualch9**
- Moves the options from the More Options section into a tab bar menu
- Moves toolbar actions into tab bar menu
- Sort appears in the main menu, Filter appears in a separate menu. When routes are being filtered, the icon changes to a selection state (filled).

| iOS 14 | iOS 15 |
| -- | -- |
|  |  |

### CODE REVIEW

File: OBAKit/Stops/StopPreferencesViewController.swift
Comment (ualch9) [inline OBAKit/Stops/StopPreferencesViewController.swift:31]: @aaronbrethorst localization strings needs to be updated to reflect this change.

**Comment by ualch9** [comment]
- Added data attribution footer

| Light | Dark |
| -- | -- |
|  |  |

File: OBAKit/Stops/StopPreferencesViewController.swift
Comment (aaronbrethorst) [reply OBAKit/Stops/StopPreferencesViewController.swift:31]: Assuming I set it up correctly, Transifex monitors pushes to `master` and will automatically pull in new/updated strings.

File: OBAKit/Stops/StopViewController.swift
Comment (aaronbrethorst) [inline OBAKit/Stops/StopViewController.swift:266]: does this suggest that there should be an `availability` check for iOS 15?

File: OBAKit/Stops/StopViewController.swift
Comment (aaronbrethorst) [inline OBAKit/Stops/StopViewController.swift:364]: Is this title shown anywhere? Should this be localized?

File: OBAKit/Stops/StopViewController.swift
Comment (aaronbrethorst) [inline OBAKit/Stops/StopViewController.swift:266]: does this suggest that there should be an `availability` check for iOS 15?

File: OBAKit/Stops/StopViewController.swift
Comment (aaronbrethorst) [inline OBAKit/Stops/StopViewController.swift:322]: should this be localized?

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
Looks great! I have a couple questions for you before merging.

File: OBAKit/Stops/StopViewController.swift
Comment (ualch9) [reply OBAKit/Stops/StopViewController.swift:364]: doesn't need to be localized. I don't think this is user visible, except maybe in catalyst. The titles are mostly comments for me, in case of future expandability.

Comment (aaronbrethorst) [reply OBAKit/Stops/StopViewController.swift:364]: Right on.

**Comment by aaronbrethorst** [comment]
@ualch9 

> Added data attribution footer

I think this is neat.

**Comment by ualch9** [comment]
oop don't merge yet, there might be a bug with the filter button

**Comment by ualch9** [comment]
@aaronbrethorst ok, ready for merge.

**Pull request merged**
